src/components/ContentListTable/ContentListChips.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useContext } from 'react';
+import { Button, Chip, ChipGroup } from '@patternfly/react-core';
+import { ContentListContext } from './ContentListContext';
+
+const ContentListChips = () => {
+
+    const context = useContext(ContentListContext)
+
+    const deleteItem = (id: string, chips, setChips) => {
+        const copyOfChips = [...chips];
+        const filteredCopy = copyOfChips.filter(chip => chip !== id);
+        setChips(filteredCopy);
+    };
+
+    const clearFilters = () => {
+        context.setSelectedVersions([])
+        context.setSelectedArches([])
+        context.setSearchQuery('')
+    }
+
+    return (
+        <>
+            <ChipGroup categoryName={'Version'}>
+                {context.selectedVersions.map(version => (
+                    <Chip key={version} onClick={() => deleteItem(version, context.selectedVersions, context.setSelectedVersions)}>
+                        {version}
+                    </Chip>
+                ))}
+            </ChipGroup>
+            <ChipGroup categoryName={'Architecture'}>
+                {context.selectedArches.map(arch => (
+                    <Chip key={arch} onClick={() => deleteItem(arch, context.selectedArches, context.setSelectedArches)}>
+                        {arch}
+                    </Chip>
+                ))}
+            </ChipGroup>
+                {(context.searchQuery !== '') && <ChipGroup categoryName={'Name/URL'}>
+                    <Chip key={'search_chip'} onClick={() => context.setSearchQuery('')}>
+                        {context.searchQuery}
+                    </Chip>
+            </ChipGroup>}
+            {(context.debouncedSearchQuery !== ''
+                && context.searchQuery !== ''
+                || context.selectedVersions.length !== 0
+                || context.selectedArches.length !== 0
```

Comment:
you can use the double bang here to say not 0, as javascript considers 0 a falsey value. So: 
!!context.selectedArches?.length